### ISSUE TITLE
Reprojection of DXF

### ISSUE THREAD

**Issue by nancylee713** [mozman/ezdxf#231]
Hello,

I encountered an issue with reading a DXF file that was reprojected from `EPSG:5174` to `EPSG:32652`.

I used the following gdal cli utility to reproject the file:

```
ogr2ogr -f "DXF" -s_srs "EPSG:5174" -t_srs "EPSG:32652" <output.dxf> <input.dxf>
```

The original file was read without problem, but when I fed the reprojected file to ezdxf, I got the following error:

```
    doc = ezdxf.readfile(filename)
  File "/usr/local/lib/python3.8/dist-packages/ezdxf/filemanagement.py", line 138, in readfile
    doc = read(fp)
  File "/usr/local/lib/python3.8/dist-packages/ezdxf/filemanagement.py", line 85, in read
    return Drawing.read(stream)
  File "/usr/local/lib/python3.8/dist-packages/ezdxf/document.py", line 252, in read
    return cls.load(tag_loader)
  File "/usr/local/lib/python3.8/dist-packages/ezdxf/document.py", line 266, in load
    doc._load(tag_loader)
  File "/usr/local/lib/python3.8/dist-packages/ezdxf/document.py", line 282, in _load
    self._load_section_dict(sections)
  File "/usr/local/lib/python3.8/dist-packages/ezdxf/document.py", line 336, in _load_section_dict
    self._2nd_loading_stage()
  File "/usr/local/lib/python3.8/dist-packages/ezdxf/document.py", line 373, in _2nd_loading_stage
    cmd = entity.post_load_hook(self)
  File "/usr/local/lib/python3.8/dist-packages/ezdxf/entities/dictionary.py", line 398, in post_load_hook
    super().post_load_hook(doc)
  File "/usr/local/lib/python3.8/dist-packages/ezdxf/entities/dictionary.py", line 148, in post_load_hook
    for k, v in list(items()):
  File "/usr/local/lib/python3.8/dist-packages/ezdxf/entities/dictionary.py", line 142, in items
    for key, handle in self.items():
  File "/usr/local/lib/python3.8/dist-packages/ezdxf/entities/dictionary.py", line 197, in items
    yield key, self.get(key)  # maybe handle -> DXFEntity
  File "/usr/local/lib/python3.8/dist-packages/ezdxf/entities/dictionary.py", line 420, in get
    assert self._default is not None and self._default.is_alive
```

I am not sure how to get past this error message and would really appreciate any pointer.

**Comment by mozman** [comment]
Please use the new `recover` module from ezdxf v0.14:

```Python
from ezdxf import recover
doc, auditor = recover.readfile(filename)
auditor.print_error_report()
auditor.print_fixed_errors()

```

Please send/post the (zipped) <output.dxf> DXF file.

**Comment by mozman** [comment]
Hi!

Thank you for the test data:

The input file `dxf_19348.dxf` is now read- and writable by ezdxf but has 19.431 recoverable errors!

The reprojected file `dxf_19348_reproj.dxf` is not readable by TrueView 2020 (AutoCAD) because of invalid table names (layers?), BricsCAD can load the file and ezdxf can load and write the file but requires the option `doc = ezdxf.readfile(filename, errors="ignore")` to replace invalid unicode characters, after saving with ezdxf, TrueView can open the file.

The `recover.readfile()` function reports additional ~34.000 unicode decoding errors.

Overall this files are not in a good shape ;-). The `dxf_19348_reproj.dxf`file is a DXF R2007 file, therefore the whole DXF file should be `utf8` encoded.

The ezdxf fixes are the `stable` and the `master` branch.

**Comment by nancylee713** [comment]
Thank you for diagnosing the test files in detail!

When a given dxf file is recovered and loaded, the output image renders all white lines. 
Is `recover` supposed to retain the original color or is it lost during the recovery process?

**Comment by mozman** [comment]
Sorry, I did some tests but I have no idea why some LWPOLYLINE entities lost its color and your files are too big to run them in a debugger!

**Comment by mozman** [comment]
I found an error in the reprojected DXF file created by ogr2ogr:

```
  0
LWPOLYLINE
  5
43C18
100
AcDbEntity  <<< here is the correct subclass for color
  8
강Ramp-E__도로경계선
100
AcDbPolyline
 70
0
 90
2
 62  <<< this is the group code for color and should be in subclass AcDbEntity!
1
 38
0
 10
336165.13218455
 20
...
```

BricsCAD can recover the color value but ezdxf can not do this.

TrueView 2020 does not open the file at all, and I don't have AutoCAD to test if AutoCAD can recover such errors.

**Comment by mozman** [comment]
The current version in the master branch can recover this misplaced DXF attributes, but will only be released with v0.15, more testing is required.

**Comment by nancylee713** [comment]
Thank you for your help!
